packages/workbox-webpack-plugin/test/node/e2etest.js

```diff
@@ -140,6 +134,69 @@ describe(`Tests for webpack plugin`, function() {
   {
     "url": "styles/stylesheet-2.css",
     "revision": "\\w*"
+  },
+  {
+    "url": "webpackEntry.js",
+    "revision": "\\w*"
+  }
+\\]\\);`);
+      const result = regex.exec(swContents);
+      expect(result).to.exist;
+    });
+  });
+
+  it('should append publicPath to webpack generated files', () => {
+    fsExtra.copySync(
+      path.join(__dirname, '..', '..', '..', 'workbox-cli', 'test', 'static', 'example-project-1'),
+      tmpDirectory);
+
+    process.chdir(tmpDirectory);
+
+    const webpackConfig = {
+      entry: {
+        webpackEntry: path.join(tmpDirectory, 'webpackEntry.js'),
+      },
+      output: {
+        path: tmpDirectory,
+        filename: '[name].js',
+        publicPath: '/public',
```

Comment:
As per https://webpack.js.org/configuration/output/#output-publicpath, the `publicPath` value is just used as a string prefix, and should end in `/` to be a representative test case.